src/redux/store/store.ts

```diff
@@ -11,12 +11,14 @@ const transformCacheVoteMap = createTransform(
   (inboundState:any) => ({ 
     ...inboundState, 
     votes : Array.from(inboundState.votes),
-    comments : Array.from(inboundState.comments)
+    comments : Array.from(inboundState.comments),
+    quickComments : Array.from(inboundState.quickComments),
   }),
   (outboundState) => ({ 
     ...outboundState, 
     votes:new Map(outboundState.votes),
-    comments:new Map(outboundState.comments)
+    comments:new Map(outboundState.comments),
+    quickComments: new Map(outboundState.quickComments)
```

Comment:
instead of `quickComments`, name them `drafts`.... moving forward this object will also maintain post drafts as well, it will also avoid confusion as well....